File: gto-support-okta/src/main/java/com/okta/authfoundation/client/dto/OidcUserInfoInternals.java
package com.okta.authfoundation.client.dto;

import com.okta.authfoundation.claims.ClaimsProvider;

class OidcUserInfoInternals {
    static OidcUserInfo create(ClaimsProvider claimsProvider) {
        return new OidcUserInfo(claimsProvider);
    }
}

File: gto-support-okta/src/main/kotlin/com/okta/authfoundation/client/dto/OidcUserInfoInternals.kt
package com.okta.authfoundation.client.dto

import com.okta.authfoundation.claims.ClaimsProvider

@Suppress("ktlint:standard:function-naming")
internal fun OidcUserInfo(claimsProvider: ClaimsProvider) = OidcUserInfoInternals.create(claimsProvider)
